spec/controllers/idv/in_person/ssn_controller_spec.rb

```diff
@@ -24,75 +24,26 @@
     allow(subject).to receive(:pii_from_user).and_return(pii_from_user)
     allow(subject).to receive(:flow_session).and_return(flow_session)
     stub_sign_in(user)
-    subject.user_session['idv/in_person'] = flow_session
```

Comment:
Is this still needed?

Reply:
running the test with and without it both work so my thought is it might not really be needed